tools/docker/scripts/build_cp2k_cmake.sh

```diff
@@ -191,7 +191,7 @@ elif [[ "${PROFILE}" == "minimal" ]] && [[ "${VERSION}" == "ssmp" ]]; then
     -DCP2K_USE_VORI=OFF \
     -DCP2K_USE_COSMA=OFF \
     -DCP2K_USE_DLAF=OFF \
-    -DCP2K_USE_SPGLIB=OFF \
+    -DCP2K_USE_SPGLIB=ON \
```

Comment:
This is the minimal build which is supposed to be - well minimal ;-)
Please enable it instead above for the following case:
```elif [[ "${PROFILE}" == "toolchain" ]] && [[ "${VERSION}" == "ssmp" ]]; then```

Reply:
Sorry, I looked at the wrong branch in this case. Looks like the `ssmp` version with the `toolchain` is already enabled. Sorry about the noise.

Reply:
NP, I also missed this. Yes, spglib already works everywhere except on Ubuntu:

https://github.com/cp2k/cp2k/blob/f8b3abcd3ac413146004a9ebf8d71317c9449627/tools/docker/scripts/build_cp2k_cmake.sh#L138-L139